examples/pytorch/translation/run_translation.py

```diff
@@ -378,7 +378,7 @@ def main():
             token=model_args.token,
         )
     # See more about loading any type of standard or custom dataset (from files, python dict, pandas DataFrame, etc) at
-    # https://huggingface.co/docs/datasets/loading_datasets.html.
+    # https://huggingface.co/docs/datasets/v2.13.1/loading.
```

Comment:
Would be nice to not ping a specific version here, as the link will be quickly outdated.

Reply:
Done I think. thanks